src/display/api.js

```diff
@@ -907,9 +910,10 @@ class PDFDocumentProxy {
  * @alias PDFPageProxy
  */
 class PDFPageProxy {
-  constructor(pageIndex, pageInfo, transport, pdfBug = false) {
+  constructor(pageIndex, pageInfo, transport, ownerDocument, pdfBug = false) {
     this._pageIndex = pageIndex;
     this._pageInfo = pageInfo;
+    this._document = ownerDocument;
```

Comment:
Please keep the `ownerDocument` name here, given the updated `DOMCanvasFactory` signature (see comment above).